setup.py

```diff
@@ -351,6 +356,8 @@ def add_dir_name(dir_name, filenames):
     filename = swig_path.lib_filename
     if platform.system() == 'Linux':
```

Comment:
platform.system() is better, as it's "windows" rather than just "nt" for os.name which isn't very clear